doc/man1/enc.pod

```diff
@@ -103,6 +104,12 @@ the B<-pass> argument.
 Use the specified digest to create the key from the passphrase.
 The default algorithm is sha-256.
 
+=item B<-iter count>
+
+Use a given number of iterations on the password in deriving the encryption key.
+High values increase the time required to brute-force the resulting file.
+This option enables the use of PBKDF2 algorithm to derive the key.
```

Comment:
Yes, it could. Although we do that now with the EVP_BytesToKey approach. Perhaps we should disable "nosalt" if PBKDF2 is used. On the other hand, if we have a proper params section at the start of the file, then we can just generate an IV normally using `RAND_bytes()`, and put the IV in there.